storybook/src/docs/hooks/useAsyncOptionsProps/useAsyncOptionsPropsSelect.stories.tsx

```diff
@@ -23,14 +23,16 @@ storiesOf("stories/hooks/useAsyncOptionsPropsSelect", module).add("Select", () =
 
     return (
         <div style={{ minHeight: "50px" }}>
-            <Form onSubmit={() => {}} initialValues={{ selectAsync: { value: "strawberry", label: "Strawberry" } }}>
+            <Form onSubmit={() => {}} keepDirtyOnReinitialize initialValues={{ selectAsync: { value: "strawberry", label: "Strawberry" } }}>
```

Comment:
Please try to fix this differently, either bei memo'ing the initialValues, or declaring the options objects in a variable and using the same object for the initial value. See https://github.com/vivid-planet/comet/blob/main/storybook/src/docs/form/components/stories/FinalFormFields.stories.tsx#L69-L84